recipe/meta.yaml

```diff
@@ -1,17 +1,20 @@
+{% set pypi = "libarchive-c" %}
+{% set name = "python-libarchive-c" %}
 {% set version = "2.8" %}
+{% set sha256 = "06d44d5b9520bdac93048c72b7ed66d11a6626da16d2086f9aad079674d8e061" %}
 
 package:
-  name: python-libarchive-c
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/l/libarchive-c/libarchive-c-{{ version }}.tar.gz
-  sha256: 06d44d5b9520bdac93048c72b7ed66d11a6626da16d2086f9aad079674d8e061
+  url: https://pypi.io/packages/source/{{ pypi[0] }}/{{ pypi }}/{{ pypi }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
   patches:
     - 0001-Add-zstd-support.patch
 
 build:
-  number: 1002
+  number: 1003
 
 requirements:
   host:
```

Comment:
can we add some runtime requirements that ensure we have binutils or a lightweight version of it around?

Reply:
:+1: